akka-cluster-sharding-typed/src/test/java/jdocs/akka/cluster/sharding/typed/HelloWorldPersistentEntityExample.java

```diff
@@ -124,8 +128,12 @@ int numberOfPeople() {
     public static final EntityTypeKey<Command> ENTITY_TYPE_KEY =
         EntityTypeKey.create(Command.class, "HelloWorld");
 
-    public HelloWorld(ActorContext<Command> ctx, String entityId) {
-      super(ENTITY_TYPE_KEY, entityId);
+    public static Behavior<Command> create(String entityId) {
+      return Behaviors.setup(context -> new HelloWorld(context, entityId));
+    }
+
+    private HelloWorld(ActorContext<Command> context, String entityId) {
+      super(ENTITY_TYPE_KEY.persistenceIdFrom(entityId));
```

Comment:
This would be ok for Akka, but not for Play since the purpose was to hide the EntityTypeKey, so move on to the Account example instead...